decidim/decidim
# Remove filter_link_params from links

#### :tophat: What? Why?

Fixes a bug that we have in Metadecidim with v0.26, regarding the caching of cards and how the "Back to list" feature works until now. Basically the problem is that for providing this feature, we polluted the params on lots of cards links, and this doesn't play nicely with caching.

So, as a fix we reimplemented this feature client side (with Javascript), so it doesn't get into the cache. As a side effect, we've lost the ugly URLs on cards when the visitor starts playing with the filters. For  saving we use the `sessionStorage` API, and for restoring we reuse the "Back to list" link.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #6432
- Fixes #8947 

#### Testing

1. Enable cache in the installation
2. Go to the proposals list page 
3. Start browsing (changing the sort order, filters, etc) 
4. Click in a proposal
5. Click in the "Back to list" link
6. See that you go back to your filtered proposals index page 

:hearts: Thank you!
